kolibri/core/auth/test/test_api.py

```diff
@@ -1147,6 +1153,37 @@ def test_session_update_last_active(self):
         new_expire_date = self.client.session.get_expiry_date()
         self.assertLess(expire_date, new_expire_date)
 
+    def test_case_insensitive_matching_usernames(self):
+        response_user1 = self.client.post(
+            reverse("kolibri:core:session-list"),
+            data={
+                "username": "shared_username",
+                "password": DUMMY_PASSWORD,
+                "facility": self.facility.id,
+            },
+            format="json",
+        )
+
+        # Assert the expected behavior based on the application's design
+        self.assertEqual(response_user1.status_code, 200)
+
+        response_user2 = self.client.post(
```

Comment:
What is different between each of these requests? Should the second one be the title case username instead?